lib/ansible/modules/cloud/azure/azure_rm_loadbalancer.py

```diff
@@ -189,11 +189,13 @@
                     - Frontend port numbers must be unique across all rules within the load balancer.
                     - Acceptable values are between 0 and 65534.
                     - Note that value 0 enables "Any Port"
+                required: True
             backend_port:
                 description:
                     - The port used for internal connections on the endpoint.
                     - Acceptable values are between 0 and 65535.
                     - Note that value 0 enables "Any Port"
+                required: True
```

Comment:
backend port is not required

Reply:
Well, the documentation should align the spec. In the spec the backend_port in inat is required but not in lb rules

https://github.com/ansible/ansible/blob/00884581ffabed74ce9ea3ee9f99a7d34cfffd56/lib/ansible/modules/cloud/azure/azure_rm_loadbalancer.py#L461-L463